### ISSUE TITLE
Linter add option to fail with warnings

### ISSUE THREAD

**Issue by diogoguimaraes** [realm/SwiftLint#160]
Is it possible to have the linter fail if there are any warnings?
I'm using travis ci and I would like to have the builds fail if there are warnings.
Thanks.

**Comment by kevindelord** [comment]
I just tried to configure Xcode to treat warnings as errors. It didn't do anything:
```
GCC_TREAT_WARNINGS_AS_ERRORS = YES
```

**Comment by jpsim** [comment]
There's a `--strict` flag that does just that!
